### DISCUSSION TITLE
Volunteers - Any specific way to become a volunteer or reach a level?

### DISCUSSION THREAD

**Discussion by esthor** [ohmyzsh/ohmyzsh#11110] [Help] [answered]
Hello, I was looking at the [Volunteers doc](https://github.com/ohmyzsh/ohmyzsh/wiki/Volunteers), and was wondering if there is a way to apply or something for the different levels?

I'm interested in helping more, but I'm not sure where to go after reading this doc. It says to open a Discussion, so that's why I'm here. (I also didn't see any specific channel on the Discord either.)

On a meta level, I'm also open to helping make the volunteer doc and leveling more robust if that's something on the table. 😸

**Comment by carlosala** [comment, marked as answer]
Hi @esthor!
Woow, thanks for your interest! For the first levels, you can just help us reviewing PRs, issues, and that kind of stuff and ping @mcornella or me to do the final review! You can also help answering discussions, Discord questions!
Thanks for your help!

**Reply by esthor** [reply]
Thanks! I'm already starting to tag you now 😄
